06_functions/main.go

```diff
@@ -74,42 +73,64 @@ Note: functions can be defined in any order in the file.
 
 func main() {
 	sayHello("John")
+	fmt.Println(diff(1, 9))
+	fmt.Println(diff(9, 1))
+	fmt.Println(diff(4, -10))
+	printInput("안녕", 3)
+	printInput("go", 7)
+	printInput("ça va", 4)
+}
 
-	// For each exercise, call the functions a couple of times with different
-	// values and print the results.
+// For each exercise, call the functions a couple of times with different
+// values and print the results.
 
-	// TODO: Write a function that takes two integers as parameters and returns
-	// the difference of the two integers.
+// TODO: Write a function that takes two integers as parameters and returns
+// the difference of the two integers.
+func diff(a int, b int) int {
+	var sum int = b - a
+	return sum
```

Comment:
Just a name suggestion

```suggestion
	var diff int = b - a
	return diff
```

Reply:
I tried to directly return actually, without variable declaration, but didn't find a compliant way.
You don't see an issue with the variable having the same name as the function, as it's inside the function I guess, okay sure.
I was thinking about converting the int to a string for example.

Reply:
conversion between types is usually a bad idea. Especially if you have the same entry/exit types, don't try converting things to strings.

Does this not work?

```go
func diff(a int, b int) int {
  return a - b
}
```

Reply:
Somehow it works. Not sure what I did before.